packages/ripple-keypairs/package.json

```diff
@@ -17,10 +17,9 @@
     "test": "test"
   },
   "dependencies": {
-    "bn.js": "^5.1.1",
-    "brorand": "^1.0.5",
-    "elliptic": "^6.5.4",
-    "hash.js": "^1.0.3",
+    "@noble/curves": "^1.0.0",
+    "@noble/hashes": "^1.0.0",
+    "brorand": "^1.1.0",
```

Comment:
use `import { randomBytes } from "@noble/hashes/utils"` instead.

Reply:
Oh, for some reason that wasn't working iirc ! 
I wonder if it was another jest test environment globals related issue though ?? Similar to https://github.com/paulmillr/noble-hashes/issues/48

Will check

Reply:
```
ripple-keypairs: FAIL ripple-keypairs test/api.test.ts (5.698 s)
ripple-keypairs:   ● api › generateSeed - secp256k1, random
ripple-keypairs:     crypto.getRandomValues must be defined
ripple-keypairs:       2 |
ripple-keypairs:       3 | export default function randomBytes(n: number) {
ripple-keypairs:     > 4 |   return rand(n)
ripple-keypairs:         |              ^
ripple-keypairs:       5 | }
ripple-keypairs:       6 |
ripple-keypairs:       at randomBytes (../../node_modules/@noble/hashes/src/utils.ts:202:9)
ripple-keypairs:       at randomBytes (src/random.ts:4:14)
ripple-keypairs:       at Object.generateSeed (src/index.ts:28:18)
ripple-keypairs:       at Object.<anonymous> (test/api.test.ts:16:22)
```

@paulmillr 
This is under Node 14 

Tests pass locally for me with Node 18

Reply:
I think this was because of node.js 14 - which will be officially unsupported in 4 days (https://endoflife.date/nodejs).

v14 doesn't support webcrypto.

Reply:
If that's acceptable for the repository, I suggest to stop supporting v14, because there will be no security updates for it from nodejs. Which will put users at risk.

Reply:
Thanks! Yeah that makes sense, I've moved to 18+ for my projects.
I can't make that call here. 

@mvadari ?

Reply:
I updated the GA file to use 16,18,20 and all the tests pass

Reply:
@mvadari @ckniffen 

Node 14 is officially unsupported now